test/unit/test_expid_command.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2015-2025 Earth Sciences Department, BSC-CNS
+#
+# This file is part of Autosubmit.
+#
+# Autosubmit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Autosubmit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for ``autosubmit expid``."""
```

Comment:
Oh, nearly forgot, I have a post-it note to check later if this needs to be moved… I **think** the file docstrings must go after the import list, but I don't recall if it's before or after :sweat_smile: don't worry as I think we have both in our code base -- and not sure if linters detect this issue.

Reply:
If you want to test it to confirm, here's one way to try it, I think (not tested):

```bash
$ python3
> import autosubmit.somepackage
> print(autosubmit.somepackage.__doc__)
```

From what I recall, that top-most comment should become the `__doc__` of the module, and be available/used in Sphinx and other tools that read docstrings like IDEs.

If it's empty, then move it to after the imports. Try again, and if it now show it, then that's where it must be.

Or perhaps check some pep in Python docs to confirm where it must be located. At least that's what I'm planning to try once I find time.